app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -21,19 +21,21 @@
   <%= submit_tag t('forms.buttons.submit.default'), class: 'btn btn-primary align-top sm-col-6 col-12' %>
 
   <br/><br/>
-  <%= link_to(t('links.two_factor_authentication.get_another_code'),
-              otp_send_path(otp_delivery_selection_form:
-                { otp_delivery_preference: @presenter.otp_delivery_preference, resend: true }),
-              class: 'btn btn-link btn-border ico ico-refresh text-decoration-none',
-              form_class: 'inline-block') %>
+  <div class="flex flex-row flex-align-center flex-wrap">
+    <%= link_to(t('links.two_factor_authentication.get_another_code'),
+    otp_send_path(otp_delivery_selection_form:
```

Comment:
our style is to indent these arguments to match the indentation of the previous line, also might be easier to break the hash on to separate lines:

```ruby
    <%= link_to(t('links.two_factor_authentication.get_another_code'),
                otp_send_path(otp_delivery_selection_form: {
                  otp_delivery_preference: @presenter.otp_delivery_preference,
                  resend: true
                },
                class: 'btn btn-link btn-border ico ico-refresh text-decoration-none flex-no-shrink margin-right-205',
                form_class: 'inline-block') %>
```